### PR TITLE
00-Introduction REFACTOR

### PR DESCRIPTION
**Pull request by volbap**
- `00-introduction.md` document restructuring. 
- Removed redundant stuff from `01-login.md` and `02-custom-login.md` documents.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/native-platforms/ios-swift/00-introduction.md [hunk 1]
```diff
@@ -5,30 +5,156 @@ description: Brief introduction to the iOS-Swift tutorials.
...
+This seed project can be useful as a starting point for the app to which you need to integrate authentication.
 
-### Done!
 
-You're all set up to start integrating Auth0 in your app. The tutorial guides contained in this section will teach you the different things you can do with it. It's recommendable that you follow them sequentially in order, as they are progressive and there is content that usually needs to be learned in previous tutorials.
\ No newline at end of file
+
+## Sample Projects
+
+Each tutorial in the serie includes a link to its corresponding sample project, which expose how to achieve the goal from the tutorial in question. You can check out all the samples [here](https://github.com/auth0-samples/auth0-ios-swift-sample/).
```
Comment (tenmaster) [inline articles/native-platforms/ios-swift/00-introduction.md]: A few thing here @pablolvillar ...
+in the serie... "isn't in the series"?
+which expose how to achieve the goal from the tutorial in question. > which expose how to achieve the tutorial's goal... (in question... doesn't sound natural here but you can keep it)

**Comment by tenmaster** [comment]
+1 @mpaktiti @chenkie

File: articles/native-platforms/ios-swift/00-introduction.md [hunk 2]
```diff
@@ -5,30 +5,156 @@ description: Brief introduction to the iOS-Swift tutorials.
...
+
+
+## Dependencies
+
+Each tutorial will require you to use either [Lock](https://github.com/auth0/Lock.iOS-OSX) or the [Auth0.swift](https://github.com/auth0/Auth0.swift) toolkit, or both. 
+
+Briefly speaking about them:
+
+- [**Lock**](https://github.com/auth0/Lock.iOS-OSX) is a widget that is easy to present in your app. It contains default templates (that can be a bit customized) for login with email/password, sign up, social providers integration, and also password recovery.
+- [**Auth0.swift**](https://github.com/auth0/Auth0.swift) is a toolkit that lets you communicate with many of the basic [Auth0 API](https://auth0.com/docs/api) functions in a neat way.
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/00-introduction.md]: @pablolvillar The Auth0 API URL should be `/api/info`

File: articles/native-platforms/ios-swift/00-introduction.md [hunk 3]
```diff
@@ -5,30 +5,156 @@ description: Brief introduction to the iOS-Swift tutorials.
...
+
+#### 1. Create an application
+
+<%= include('../../_includes/_new_app') %>_
+
+![App Dashboard](/media/articles/angularjs/app_dashboard.png)
+
+#### 2. Configure your callback URLs
+
+<%= include('../_includes/_callback-url-introduction') %>
```
Comment (mpaktiti) [inline articles/native-platforms/ios-swift/00-introduction.md:82]: @pablolvillar This does not exist and as a result the page cannot be displayed.

Comment (volbap) [inline articles/native-platforms/ios-swift/00-introduction.md:82]: @hzalaz I took that line from the previous version of the iOS quickstart. Do you have any idea how can I get that file, or re-generate it?

**Comment by volbap** [comment]
Closing this pull request. These changes will be included in a new pull request.

**Pull request closed without merging**